src/components/common/TmBalance.vue

```diff
@@ -147,6 +146,22 @@ export default {
           // query if successful or not as even an unsuccessful tx costs fees
           refetchNetworkOnly(this.$apollo.queries.overview)
         }
+      },
+      userRewardsIncremented: {
```

Comment:
isn't this happening every block if the user has delegations?

Reply:
not every block I think, but nearly yes